src/ducks/components/OneAppLoading.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { Component } from 'react'
+import withBreakpoints from 'cozy-ui/react/helpers/withBreakpoints'
+
+export const Placeholder = ({ width, height, autoMargin, style }) => {
+  const widthStyle = Array.isArray(width)
+    ? `${Math.random() * (width[1] - width[0]) + width[0]}rem`
+    : width
+  if (style == undefined) style = {}
+  style = {
+    ...style,
+    width: widthStyle,
+    height
+  }
+  if (autoMargin) style.margin = 'auto'
+  return <span className="sto-sections-placeholder" style={style} />
+}
+
+export class AppsLoading extends Component {
```

Comment:
You may want to rename the component to match correctly this file.
Also, could we have a screenshot of this one app loading please? It will be better to see directly what it looks like, thanks :)

Reply:
![image](https://user-images.githubusercontent.com/7733019/48408413-8ed0c980-e739-11e8-9db9-20d7c086da0a.png)